.github/workflows/build.yml

```diff
@@ -36,8 +45,29 @@ jobs:
       with:
         name: Linux flatc binary ${{ matrix.cxx }}
         path: flatc
+    # Below if only for release.
+    - name: Zip file
+      if: startsWith(github.ref, 'refs/tags/')
```

Comment:
i.e., change to these to `if: true` or the equivalent.

Reply:
I can do that. The limitation is that the provenance info won't be uploaded to a release, since it will run on a PR. Is that OK?

Reply:
Sure, i just wanted to make sure the steps run without error.

Reply:
done, PTAL. I've kept the one for the last job (`slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.2.0`) because generation on PR is currently not supported, so it will fail.

Reply:
sorry, the release upload will fail too. Let me update real quick.

Reply:
Looks like you didn't push

Reply:
Update done. I kept the if statement for the call to `uses: softprops/action-gh-release@v1`, because I think the upload would fail on a PR - it expected a tag.

Reply:
You're right. Pushed now

Reply:
Looks like it ran OK, feel free to revert the testing change and we can get it in

Reply:
Done, PTAL. I may be beneficial to create a draft release (using the GitHub UI) to sanity check that the zip files and the provenance are in the assets once the workflow run completes.